pcsx2/gui/Panels/PluginSelectorPanel.cpp

```diff
@@ -699,6 +701,48 @@ void Panels::PluginSelectorPanel::OnEnumComplete( wxCommandEvent& evt )
 				else if( index_sse2 >= 0 ) m_ComponentBoxes->Get(pid).SetSelection( index_sse2 );
 				else m_ComponentBoxes->Get(pid).SetSelection( 0 );
 			}
+			else if (pid == PluginId_PAD)
+			{
+				int count = (int)m_ComponentBoxes->Get(pid).GetCount();
+
+				int index_lilypad = -1;
+				int index_onepad = -1;
+				int index_onepad_legacy = -1;
+
+				for( int i = 0; i < count; i++ )
+				{
+					auto str = m_ComponentBoxes->Get(pid).GetString(i);
+
+					if (str.Contains("lilypad")) index_lilypad = i;
+					if (str.Contains("onepad"))
+					{
+						if (str.Contains("legacy"))
+							index_onepad_legacy = i;
+						else
+							index_onepad = i;
+					}
+				}
+
```

Comment:
Maybe a lilypad string check would be nice too, in case there are multiple plugins added by user on windows.

Bonus: Maybe code formatting can be adjusted to better suit the rest of the code, your call.

Reply:
Well, it's got a check for a plugin with "lilypad" in the name, and on windows, it's supposed to pick that one over any others, whereas linux picks onepad first, onepad_legacy second, lilypad third, and anything else afterwards.

I mainly wasn't sure if "lilypad" will match what we want properly on Windows, or if there's anything else it should be searching for there.

I could adjust the formatting more, but it'd probably be better just to run it through clang_format at some point with a nice set of defaults rather than fiddling with it too much here.

For that matter, if any other plugins need defaults set, we might end up wanting to turn this into a case statement later...

Reply:
Clang would be ideal indeed.

Doing the defaults for all plugins is a nice idea too but most of the time people use the default plugins that come with pcsx2 itself. Up to you to decide what you wanna do with that.